consumer/src/main/java/com/manquius/twelvefactor/client/PulsarConsumeClient.java

```diff
@@ -5,14 +5,18 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static java.util.Optional.ofNullable;
+
 
 class PulsarConsumeClient implements ConsumeClient {
 
     private final PulsarClient client;
     private final ConsumerBuilder<String> builder;
 
     public PulsarConsumeClient() throws ClientCreationException {
-        String pulsarBrokerRootUrl = "pulsar://localhost:6650";
+        String host = ofNullable(System.getenv("TWELVEFACTOR_PULSAR_PROXY_SERVICE_HOST")).orElse("localhost");
```

Comment:
Use config map here as env variable

Reply:
The used variable is automatically provided by kubernetes to the different containers.